src/liballoc/collections/btree/node.rs

```diff
@@ -687,8 +676,7 @@ impl<'a, K, V> NodeRef<marker::Mut<'a>, K, V, marker::LeafOrInternal> {
     /// Removes a key/value pair from the end of this node. If this is an internal node,
     /// also removes the edge that was to the right of that pair.
     pub fn pop(&mut self) -> (K, V, Option<Root<K, V>>) {
-        // Necessary for correctness, but this is an internal module
-        debug_assert!(self.len() > 0);
+        assert!(self.len() > 0);
```

Comment:
Could be integrated using `let idx = self.len().checked_sub(1).expect(...)` or so, but that smells of premature micro-optimization.